keras_cv/models/models_test.py

```diff
@@ -0,0 +1,109 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Integration tests for KerasCV models."""
+
+import tensorflow as tf
+from absl.testing import parameterized
+from tensorflow.keras import backend
+
+from keras_cv.models import densenet
+
+MODEL_LIST = [
+    (densenet.DenseNet121, 1024),
+    (densenet.DenseNet169, 1664),
+    (densenet.DenseNet201, 1920),
+]
+
+
+class ApplicationsTest(tf.test.TestCase, parameterized.TestCase):
+    def assertShapeEqual(self, shape1, shape2):
+        if len(shape1) != len(shape2):
+            raise AssertionError(
+                "Shapes are different rank: %s vs %s" % (shape1, shape2)
+            )
+        for v1, v2 in zip(shape1, shape2):
+            if v1 != v2:
+                raise AssertionError("Shapes differ: %s vs %s" % (shape1, shape2))
+
+    @parameterized.parameters(*MODEL_LIST)
+    def test_application_base(self, app, _):
+        # Can be instantiated with default arguments
+        model = app(
+            input_shape=(224, 224, 3), include_preprocessing=False, weights=None
```

Comment:
Potentially we can rename include_preprocessing to `include_rescaling` or something.

Reply:
I like 'include_rescaling' since its more explicit.

Reply:
just talked to Francois about this, unfortunately rescaling an image does mean resizing as well in many contexts.  Though, I also can't think of a better name.  I do think rescaling is better than preprocessing, but it is still tricky - as rescaling may make users think we resize their images

Reply:
I updated this to `include_rescaling` for now, I think this is at a minimum better than `include_preprocessing`.  preprocessing is a superset of rescaling.